lib/wallet.js

```diff
@@ -138,6 +138,7 @@ function authorizeZeroConf (settings, tx, machineId) {
   const plugin = cryptoConfig.zeroConf
   const zeroConfLimit = machineConfig.zeroConfLimit
 
+  // what if tx.fiat === undefined?
```

Comment:
I'd rather not have this kind of thing in comments (even thought there are some of these in the existing code). Better to specifically handle it.

Reply:
Yes,
sorry I meant to add a comment here.
What should be the expected behavior here in that case?

Reply:
I believe tx.fiat should never been `undefined` at this point, so you can throw an error if that happens.